src/components/loader-bar/loader-bar-test.stories.tsx

```diff
@@ -27,6 +27,6 @@ export const DefaultStory = ({ size, ...args }: LoaderBarProps) => {
 
 DefaultStory.storyName = "default";
 
-export const LoaderBarComponentTest = ({ ...args }) => {
+export const LoaderBarComponentTest = (args: LoaderBarProps) => {
```

Comment:
This is really minor, but it might be worth changing the name of the parameter here, just so there isn't any confusion between the mock components created for the Cypress tests and storybook stories.

```suggestion
export const LoaderBarComponentTest = (props: LoaderBarProps) => {
```